### DISCUSSION TITLE
4g lte modem  - add-image.excludes testimonial

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#567] [General]
Not a problem at all with ltsp, just to clarify.

first time using a 4g lte modem as internet connection on an ltsp server or any build for that matter . everything worked fine on client machines other than they would all reboot when shutting each client  down. After from 5 mins to 20 minutes the modem on server would start resetting. The only time the modem would do this resetting thing was if a pxe boot  machine was online. As soon as the client was shut off the modem had internet access again. Very much a head scratcher . 

Long story short i added the 'hooks' for the modem firmware 'mbim', and 'qmi' binaries to the add-image.excludes file, rebuilt client image and have had no problems since. I know this doesnt make sense but this is all i changed.  Also this circumvented the problem with the client machines rebooting rather than shutting down. 

Hope this may help someone else out in a future ltsp build
